.repos/repositories.yml

```diff
@@ -188,11 +188,19 @@
 - url: https://github.com/kuzzleio/sdk-dart.git
   repo_name: sdk-dart
   doc_version: 2
-  stable: master
+  stable: 2-dev
```

Comment:
```suggestion
  stable: master
```

Reply:
nope... `master` is the current stable branch for version 3 of the Dart SDK. This repo has no `X-stable` branches, so no `2-stable` branch exists...

Reply:
Oh ok so you need to create a `2-stable` branch on the Dart SDK repo 🙂

Reply:
```suggestion
  stable: 2-stable
```

Reply:
Apparently we don't need it (dixit @jenow ) so, for now, I will leave things this way (not such a problem, btw)